### PR TITLE
[FIX] LGTM recommendations

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
Fixes some LGTM recommendations:
https://lgtm.com/projects/g/nilearn/nilearn/?mode=list&severity=recommendation

Please review carefully. Not sure about these, especially unused imports. Unused imports might have undocumented  side effects. If that's the case, they should be documented when importing, as is already the case elsewhere in the source code.

### CODE REVIEW

File: nilearn/plotting/surf_plotting.py [hunk 1]
```diff
@@ -11,7 +11,6 @@
 from matplotlib.colorbar import make_axes
 from matplotlib.cm import ScalarMappable, get_cmap
 from matplotlib.colors import Normalize, LinearSegmentedColormap
-from mpl_toolkits.mplot3d import Axes3D
```
Comment (agramfort) [inline nilearn/plotting/surf_plotting.py:14]: I think this is required for old matplotlib. Rather put a `#noqa` at the end of the line

**Review by agramfort — COMMENTED** [review]
besides LGTM

thx @DimitriPapadopoulos

File: nilearn/plotting/surf_plotting.py (hunk 1 above)
Comment (DimitriPapadopoulos) [reply nilearn/plotting/surf_plotting.py:14]: Done.

File: nilearn/plotting/displays.py [hunk 2]
```diff
@@ -1910,6 +1910,7 @@ def find_cut_coords(self, img=None, threshold=None, cut_coords=None):
                                                self._cut_displayed)
         return cut_coords
 
+    @classmethod
```
Comment (jeromedockes) [inline nilearn/plotting/displays.py:1913]: this should be `staticmethod` otherwise the bound method will have the class as its first parameter. btw it is a bit confusing that some class methods in this module call their first parameter `self` rather than the more conventional `cls`, but I guess that is outside the scope of this PR

Comment (DimitriPapadopoulos) [reply nilearn/plotting/displays.py]: Ah, right, thank you.

Comment (DimitriPapadopoulos) [reply nilearn/plotting/displays.py]: See #2923 for self -> cls.

**Review by DimitriPapadopoulos — COMMENTED** [review]
Seems to be working now. Strange.

File: nilearn/_utils/niimg_conversions.py [hunk 3]
```diff
@@ -476,7 +476,7 @@ def concat_niimgs(niimgs, dtype=np.float32, ensure_ndim=None,
         lengths.append(niimg.shape[-1] if ndim == 4 else 1)
 
     target_shape = first_niimg.shape[:3]
-    if dtype == None:
+    if dtype is None:
```
[the remark this answers is not in the document]
Comment (jeromedockes) [reply nilearn/_utils/niimg_conversions.py:479]: what is the error message?

**Review by jeromedockes — APPROVED** [review]
thanks!

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
Thx @DimitriPapadopoulos

**Comment by bthirion** [comment]
Thx !
